quickstart.mdx

```diff
@@ -31,7 +31,37 @@ You can find your URL from the Overview page of the dashboard.
 
 This URL is instantly available and will update whenever you make changes to your documentation. It's perfect for testing and sharing with your team during development.
 
-## Development Workflows
+### Install the GitHub App
+
+Mintlify provides a GitHub App that automates the deployment process when you push changes to your repository. If you don't already have a GitHub account, you will need to [sign up for GitHub](https://github.com/signup).
```

Comment:
in terms of how onboarding goes, we basically require a github account for onboarding. so im not 100% sure if it makes sense to mention creating an account. but curious ur thoughts on the user flow. are people reading this before or after onboarding? or if it's sales-led it's also a different user flow

Reply:
I'm in favor of keeping it streamlined as possible, so will remove. Plus it should be clear that when using GitHub or any other third-party tool, you'll need an account with that tool